recipes-config/systemd-resource-control/services/service.brcm43438.conf

```diff
@@ -0,0 +1,2 @@
+[Service]
```

Comment:
Let's wait with BSP specific services until we have a solution to handle variants.